[DOCS] 8.1.2 Release Notes (#1778)
elastic/security-docs: docs/release-notes/8.1.asciidoc

[[release-notes-header-8.1.0]]
== 8.1

[discrete]
[[release-notes-8.1.2]]
=== 8.1.2

[discrete]
[[bug-fixes-8.1.2]]
==== Bug fixes and enhancements

> * Ensures {endpoint-sec} can continue to run on all supported Windows versions by changing the primary signer of the `elastic-endpoint.exe` file from ELASTICSEARCH B.V. to Elasticsearch, Inc. (https://github.com/elastic/endpoint/issues/15[#15]).

Reviewer: I think it would be helpful to format the signer values in some way, to clearly demarcate where each value starts and ends. I'm not sure if backticks or quotation marks (or something else) would be best, but here's an example using backticks...
```suggestion
* Ensures {endpoint-sec} can continue to run on all supported Windows versions by changing the primary signer of the `elastic-endpoint.exe` file from `ELASTICSEARCH B.V.` to `Elasticsearch, Inc.` (https://github.com/elastic/endpoint/issues/15[#15]).
```

> * Ensures {endpoint-sec} can continue to run on all supported Windows versions by changing the primary signer of the `elastic-endpoint.exe` file from `ELASTICSEARCH B.V.` to `Elasticsearch, Inc.` (https://github.com/elastic/endpoint/issues/15[#15]).

Reviewer 2: ```suggestion
* Ensures {endpoint-sec} continues to run on all supported Windows versions by changing the primary signer of the `elastic-endpoint.exe` file from `ELASTICSEARCH B.V.` to `Elasticsearch, Inc.` (https://github.com/elastic/endpoint/issues/15[#15]).
```

[discrete]
[[release-notes-8.1.1]]
=== 8.1.1

[discrete]
[[known-issue-8.1.1]]
==== Known issues

> * {endpoint-sec} cannot be run on Windows 8.1 and Server 2012 R2 (https://github.com/elastic/endpoint/issues/15[#15]).

Reviewer 2: ```suggestion
* {endpoint-sec} cannot run on Windows 8.1 or Server 2012 R2 (https://github.com/elastic/endpoint/issues/15[#15]).
```

[[release-notes-header-8.1.0]]
== 8.1

[discrete]
[[release-notes-8.1.2]]
=== 8.1.2

[discrete]
[[bug-fixes-8.1.2]]
==== Bug fixes and enhancements

> * The primary signer for `elastic-endpoint.exe` has changed from ELASTICSEARCH B.V. to Elasticsearch, Inc.

Reviewer 3: I didn't mention it in the issue, but this change only affects Windows. We do sign the macOS Endpoint as well (though it isn't named `elastic-endpoint.exe`). I don't feel strongly whether its worth calling out the OS here or leaving this as-is but am mentioning it in case you think it matters.
